### DISCUSSION TITLE
Can a watch as standalone collector spread its data directly to follower?

### DISCUSSION THREAD

**Discussion by rudiruderboot** [Navid200/xDrip#76] [Watches] [answered]
Hi everyone! 
Is a smartwatch that has been set as collector, capable of spreading its data to followers directly? Meaning without a phone in BT range that does the job.
I bought a Huawei Watch 2, got xdrip to run on it (downgraded to android wear 2.9) and succeeded in setting it up as the standalone collector of Dexcom 6 transmitter data.
And since the watch has wifi and LTE (sim card) connection, I thought it would send its data to other devices directly. But apparently it does not.
It seems, the watch needs the phone, it was set up with, in bluetooth range to do the job of spreading.

The background is, that our son is only 7 years old and he shoundn't carry a smartphone along all the time.

Does anybody know of a way to make it work?

I couldn't find options to set the watch as xDrip Sync Master or Nightscout Master. Neither in the phone's wear app, xDrip app nor on the watches xDrip settings.

Am I missing something or is it just not possible?
It would be so great to make it work. Our son would have the solution on his wrist and no lose extra item as inappropriate as a smartphone. I mean, he is a kid!

Thanks to everyone!

**Comment by Navid200** [comment]
@rudiruderboot If your son is home with his watch on, and you go for a walk with your phone in your pocket, do you not see his readings update on your phone as you walk?

**Comment by rudiruderboot** [comment, marked as answer]
Woop, got it to work.
I made a whole new setup from the scratch and maybe, I did some steps in different order.
I remember that last time, my phone was set as Sync Master by the time I gave the watch its collector rights. And whatever I did, the watch did not send a single bg value independently.
This time, another phone was the collector and Sync Master before the new start. I disconnected this other phone, startet the watch with my phone (wich was set up as Follower), gave collection permissions to the watch and - to see wether it worked or not - disabled wifi and bluetooth on my phone.
The watch sent values to my phone (using its sim card). But to my phone only, whereas other phone that where set as Followers didn't get any values. Then I made my phone the master, and now it works!

So the answer to my own questions is: Yes and no. Yes, the watch can send its data to other devices using mobile Internet (and wifi).
And no, it doesn't send to followers directly but it addresses the phone it was setup with first, and this device can be set as master AFTER THIS.

It means, that although the watch and my phone don't interact directly (because all nearfield capabilities are off, i.e. wifi, BT, nfc..) the two devices seem to be related. The wear os app on the phone might be the reason. I wonder if I can set up the other phone as sync master. The one that has no wear os app installed. I will check that tomorrow. Now I need some sleep. This kept me up many hours.

**Reply by enginemr2** [reply]
If the CGM is the Dexcom G6 it should get and relay the data every 5 minutes. I use the Sony SWR50 smartwatch as a standalone collector. It does relay the data to the phone every 5 minutes. I am only using bluetooth to save watch battery charge. Since the watch is the standalone collector I have recently discovered it can also reveal a problem with the Dexcom transmitter. Some time ago my watch battery after only 12 hours of operation dropped to only a 10% charge. It usually has over 50% after 16 hours of operation. After some time the transmitter failed. In fact only one of the G6 bluetooth ports failed, the one dedicated to smart devices. The other bluetooth port dedicated to the Dexcom receiver continued to work. I changed the transmitter and both ports worked. Also the watch again shows more than 50% charge after 16 hours of operation. As requested I sent the defective transmitter to Dexcom for analysis.

**Comment by rudiruderboot** [comment]
Plus, I wonder If the whole thing will still be working as soon as I patch the watch. Because getting values every 10 minutes only is somehow annoying. I will give updates here!

**Comment by dominolu** [comment]
你需要一个支持esim卡的智能手表，但目前华为的watch3 不支持xdrip，我也正在感觉如何在华为watch3上安装xdrip
